src/test/run-make/llvm-module-pass/Makefile

```diff
@@ -0,0 +1,9 @@
+-include ../tools.mk
+
+all:
+	$(CXX) $(LLVM_CXXFLAGS) -c llvm-pass.so.cc -o $(TMPDIR)/llvm-pass.o
+	$(AR) rcs $(TMPDIR)/libllvm-pass.a $(TMPDIR)/llvm-pass.o
+	$(RUSTC) plugin.rs --crate-name some_plugin --crate-type dylib \
+	                   -C prefer-dynamic --emit=dep-info,link \
+	                   -L native=$(TMPDIR) -l static=llvm-pass
+	$(RUSTC) main.rs --extern some_plugin=$(TMPDIR)/libsome_plugin.dylib
```

Comment:
You should be able to drop the `--extern` here because it will automatically be picked up as part of the `-L`